src/components/Tokens/TokenDetails/BalanceSummary.tsx

```diff
@@ -67,8 +67,15 @@ export default function BalanceSummary({ token }: { token: Currency }) {
   const theme = useTheme()
   const { label, color } = getChainInfo(asSupportedChain(chainId) ?? ChainId.MAINNET)
   const balance = useCurrencyBalance(account, token)
-  const formattedBalance = formatCurrencyAmount(balance, NumberType.TokenNonTx)
-  const formattedUsdValue = formatCurrencyAmount(useStablecoinValue(balance), NumberType.FiatTokenStats)
+  const { formatCurrencyAmount } = useFormatter()
```

Comment:
The Swap input panel bug with a formatted language also happens here as well
![Screenshot 2023-09-06 at 9 39 09 AM](https://github.com/Uniswap/interface/assets/11512321/74f209f9-9769-480d-aec1-e32a7a588069)
![Screenshot 2023-09-06 at 9 40 12 AM](https://github.com/Uniswap/interface/assets/11512321/d3af189e-53d6-4bee-8b93-28b2f5bbafe7)